packages/squiggle-lang/src/library/registry/core.ts

```diff
@@ -66,6 +66,11 @@ export class Registry {
     if (!this.fnNameDict.has(fnName)) {
       throw new Error(`Function ${fnName} doesn't exist in registry`);
     }
```

Comment:
`has` check above is not necessary anymore.

Reply:
fair